src/ui/component/DisciteCourse.html

```diff
@@ -30,7 +30,7 @@
     </div>
 
     <slot name="actions">
-        <DisciteCourseActions {course} {modifier} {module} />
+        <DisciteCourseActions {course} {modifier} {module} {today}/>
```

Comment:
You can just do `today={new Date()}` and then remove it everywhere else. It stops you from repeating since you're only using it one place, then you don't have to continually keep passing `today` down to the page -> component -> component.

Alternatively you can add it to datetime.ts and within DisciteCourseActions call it with $datetime if you think we are going to use it more often